Cinema-Reservation-System-Front-End-Client-Part-Internship-project/src/app/authentication/register/register.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component } from '@angular/core';
+import { ButtonComponent } from '../../shared/button/button.component';
+import { InputFieldComponent } from '../../shared/input-field/input-field.component';
+import {
+  FormBuilder,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
+import { AuthenticationService } from '../shared/authentication.service';
+
+@Component({
+  selector: 'app-register',
+  standalone: true,
+  imports: [ButtonComponent, InputFieldComponent, ReactiveFormsModule],
+  templateUrl: './register.component.html',
+  styleUrl: './register.component.css',
+})
+export class RegisterComponent {
+  buttonLabel: string = 'Register';
```

Comment:
Lets move this to the template itself because we don't change it anywhere in the component.